target/linux/lantiq/xrx200/base-files/etc/board.d/02_network

```diff
@@ -66,6 +66,9 @@ lantiq_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan" "2:lan" "4:lan" "5:lan" "6t@eth0"
 		;;
+	tplink,archer-d2)
+		ucidef_set_interface_lan "lan1 lan2 lan3 lan4"
```

Comment:
DSA?

Reply:
It is, but it requieres https://github.com/openwrt/openwrt/pull/3085 and also has some issues. Thinking about it, do I need to add WIP/RFC to the title?